modules/oe_webtools_laco_service/README.md

```diff
@@ -0,0 +1,35 @@
+# OpenEuropa Webtools Laco Service
+
+A Webtools Laco service that provides information on whether entity has a translation in a specific language.
```

Comment:
The LACO service should also work for non-entity pages, in which case we simply return all available site languages.

In Drupal 7 this feature was covered with [this test](https://github.com/ec-europa/platform-dev/blob/release-2.5/profiles/common/modules/custom/nexteuropa_laco/tests/LanguageCoverageServiceTest.php#L18).